### PR TITLE
Merge REQ-230 from feature branch to master

### PR DESCRIPTION
**Pull request by krizex**
As title.

### CODE REVIEW

File: XenAdmin/Dialogs/PropertiesDialog.cs [hunk 1]
```diff
@@ -202,6 +203,9 @@ private void Build()
                 if (is_pool_or_standalone && !Helpers.FeatureForbidden(xenObject.Connection, Host.RestrictLivePatching))
                     ShowTab(LivePatchingEditPage = new LivePatchingEditPage());
 
+                if (is_pool_or_standalone && !Helpers.FeatureForbidden(xenObject.Connection, Host.RestrictIGMPSnooping) && Helpers.GetMaster(pool).vSwitchNetworkBackend)
```
Comment (MihaelaStoica) [inline XenAdmin/Dialogs/PropertiesDialog.cs]: `vSwitchNetworkBackend` is now a method, so this needs to be changed to `vSwitchNetworkBackend()`

File: XenModel/XenAPI-Extensions/Host.cs [hunk 2]
```diff
@@ -548,6 +548,16 @@ public static bool RestrictLivePatching(Host h)
             return h._RestrictLivePatching;
         }
 
+        private bool _RestrictIGMPSnooping
```
Comment (MihaelaStoica) [inline XenModel/XenAPI-Extensions/Host.cs]: We recently converted all extension get properties of the API classes to methods, see for example here:
https://github.com/xenserver/xenadmin/pull/1757/commits/ae22560ce86f6a739e07807a6b74a321dedea7e1#diff-9249bdd29e594ac8a4520e24be0254d9L218
Please remove the `_RestrictIGMPSnooping`  property and move the code inside the `RestrictIGMPSnooping(Host h)` method.

File: XenAdmin/Dialogs/PropertiesDialog.cs (hunk 1 above)
Comment (huizh) [reply XenAdmin/Dialogs/PropertiesDialog.cs]: Updated. Thanks.

File: XenModel/XenAPI-Extensions/Host.cs (hunk 2 above)
Comment (huizh) [reply XenModel/XenAPI-Extensions/Host.cs]: Updated. Thanks.

**Comment by huizh** [comment]
Hi @krizex, could you please pick the commit https://github.com/huizh/xenadmin/commit/995378053ba96659230462cd9ee78b299965cbc8 on this PR?

Thank you.

**Comment by huizh** [comment]
Hi @MihaelaStoica, for the latest XenAPI bindings, could you please pick https://github.com/huizh/xenadmin/commit/b5b0e77532321c08b6eee6b42c38e67d78caabe3?

Thanks.

**Pull request merged**